data/avro-schema/src/main/resources/avro/net/corda/data/membership/GroupParameters.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "GroupParameters",
+  "namespace": "net.corda.data.membership",
+  "doc": "Avro representation of GroupParameters.",
+  "fields": [
+    {
+      "name": "viewOwner",
+      "doc": "For dynamic networks it is the short hash of member's holding identity or for static registration the name of the owning service.",
+      "type": ["string"]
```

Comment:
I think we'll need something else here. 

For the static registration that Yash is looking at we'll need a service identifier and a group ID. For every other case we'll need a way to connect the record to a holding identity; i.e. holding identity hash, holding identity, or just name and group identifier. 

I'd like to change `viewOwner` to either:
1) Two new fields; group identifier and owner identifier (which would be service name or x500 name)
2) One new type called something like `GroupParametersOwner` which contained group identifier and owner identifier.

What I like about `GroupParametersOwner` is that it's clearly specifically for group parameters and it is clear that it is intentionally different to `HoldingIdentity`.